<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>397</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGpxExport"/>
  </widget>
  <action name="actionUpload">
   <property name="icon">
    <iconset>
     <normaloff>icons/upload.png</normaloff>icons/upload.png</iconset>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </action>
  <action name="actionGpxExport">
   <property name="icon">
    <iconset>
     <normaloff>icons/handheld-gps-device.jpg</normaloff>icons/handheld-gps-device.jpg</iconset>
   </property>
   <property name="text">
    <string>GPX Export</string>
   </property>
   <property name="toolTip">
    <string>GPX Export</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
